cmd/kube-controller-manager/app/options/options.go

```diff
@@ -412,11 +418,30 @@ func (s KubeControllerManagerOptions) Config(allControllers []string, disabledBy
 
 	eventRecorder := createRecorder(client, KubeControllerManagerUserAgent)
 
+	// get resource provider kube configs
+	var resourceProviderClients []clientset.Interface
+	if len(s.ResourceProviderKubeConfig) > 0 {
```

Comment:
what if 0? is this case be treated already somewhere else?

Reply:
If ResourceProviderKubeConfig is not provided, fallback to single cluster case.

> what if 0? is this case be treated already somewhere else?